samples/DotNet/Microsoft.Azure.EventHubs/Rbac/CustomRole/readme.md

```diff
@@ -43,3 +36,19 @@ To run the sample, follow these steps:
 6. Update the sample with Event Hubs namespace and Storage account name.
 7. Run [Sender application](https://github.com/Azure/azure-event-hubs/tree/serkar.AddCustomRbacEhSample/samples/DotNet/Microsoft.Azure.EventHubs/SampleSender) to send some number of messages to your event hub.
 8. Run the CustomRole sample to receive those events back.
+
+**Note:** This sample uses the legacy Event Hubs library `Microsoft.Azure.EventHubs`. We strongly recommend you to use the new library `Azure.Messaging.EventHubs`. 
```

Comment:
The note should be made available much earlier before leading the user down the path of Track 1 package